generated/nidigital/session.py

```diff
@@ -2843,21 +2848,25 @@ def fetch_capture_waveform(self, site_list, waveform_name, samples_to_read, time
 
 
         Returns:
-            data (list of int):
+            waveform (list of Waveform): List of named tuples with fields:
```

Comment:
Is the order important?
If not, it might make sense for this to be a dictionary of `site`->`waveform data`
And perhaps we should consider replacing the name tuple with the same... one less type.